docs/read_the_docs/source/workflow_inputs/create_characteristics_rst.rb

```diff
@@ -164,34 +166,109 @@ def get_measure_xml(filepath)
       r_arguments << argument if !r_arguments.include?(argument)
     end
   end
-  next if r_arguments.empty?
+  if r_arguments.any?
 
-  name = 'Arguments'
+    name = 'Arguments'
+    f.puts(name)
+    f.puts('*' * name.size)
+    f.puts
+    f.puts('.. list-table::')
+    f.puts('   :header-rows: 1')
+    f.puts('   :stub-columns: 1')
```

Comment:
Are you using `stub-columns` to get the bold text, or are you also interested in the center align?

Reply:
It is the bold text for consistency. It is the only way I could think of to make the formatting match.